nixos/modules/services/databases/openldap.nix

```diff
@@ -46,6 +46,15 @@ in
         ";
       };
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.openldap;
+        description = ''
+          Package to use for OpenLDAP. This can be overridden if
+          custom modules or other similar features are necessary.
```

Comment:
Could you clarify the description some more to make it clear that we always mean an `openldap` package, just that it can be a custom **OpenLDAP** package with an override to enable specific features or modules.

Reply:
Cool, I've updated the description - what do you think?